sorbet/rbi/shims/fixture_loader.rbi

```diff
@@ -0,0 +1,15 @@
+# typed: true
+
+module Tapioca
+  module Compilers
+    module Dsl
+      class ActiveRecordFixtures < Base
+        class FixtureLoader
```

Comment:
Why do we need this shim? We should be able to generate `ActiveRecord::TestFixtures` with the proper `mixes_in_class_modules` annotation.

Reply:
Can Tapioca currently pick up Active Support special attributes invoked from within an `included` block? I think this is something we're currently missing.

[This is where the methods are created](https://github.com/rails/rails/blob/main/activerecord/lib/active_record/test_fixtures.rb#L19-L27).